### PR TITLE
Update French Translations + Correcting typos

### PR DESCRIPTION
**Pull request by Razikd29**

### CODE REVIEW

**Comment by Razikd29** [comment]
ok, it should be good, I can't see what's missing or blocking the checks 🤔

**Comment by Aegyo** [comment]
Do you have a details button next to the failed check?

It is complaining about Gigantamax Machop missing from the French file

**Comment by Razikd29** [comment]
Ah right ! I deleted it since there's no Gigantamax Machop

File: locales/fr/pokemon.json [hunk 1]
```diff
@@ -1304,116 +1303,116 @@
...
   "Stonjourner": "Dolman",
   "Eiscue": "Bekaglaçon",
   "Eiscue (Ice Face)": "[[Eiscue]] (Tête de Gel)",
-  "Eiscue (No Ice Face)": "",
+  "Eiscue (No Ice Face)": "[[Eiscue]] (Tête Dégel)",
   "Indeedee": "Wimessir",
-  "Indeedee (Male)": "[[Indeedee]] (Mâle)",
-  "Indeedee (Female)": "",
+  "Indeedee (Male)": "[[Indeedee]] [[alt.male]]",
+  "Indeedee (Female)": "[[Indeedee]] [[alt.frmale]]",
```
Comment (Aegyo) [inline locales/fr/pokemon.json:1380]:
```suggestion
  "Indeedee (Female)": "[[Indeedee]] [[alt.female]]",
```

File: locales/fr/pokemon.json [hunk 2]
```diff
@@ -1304,116 +1303,116 @@
...
   "Zacian (Battle Hero)": "[[Zacian]] (Héro Aguerri)",
-  "Zacian (Crowned Sword)": "",
+  "Zacian (Crowned Sword)": "[[Zacian]] (Épée Suprême)",
   "Zamazenta": "Zamazenta",
   "Zamazenta (Battle Hero)": "[[Zamazenta]] (Héro Aguerri)",
-  "Zamazenta (Crowned Shield)": "",
+  "Zamazenta (Crowned Shield)": "[[Zamazenta]] (Bouclier Suprême)",
   "Eternatus": "Éthernatos",
-  "Eternamax Eternatus": "",
+  "Eternamax Eternatus": "[[Etenatus]] Infinimax",
```
Comment (Aegyo) [inline locales/fr/pokemon.json:1402]:
```suggestion
  "Eternamax Eternatus": "[[Eternatus]] Infinimax",
```

**Comment by Razikd29** [comment]
Should I add another commit with all Machop gigantamax being removed ?

**Review by Aegyo — CHANGES_REQUESTED** [review]
Couple of spelling misakes, + the missing g-max machop

**Comment by Aegyo** [comment]
> Should I add another commit with all Machop gigantamax being removed ?

Yep, that would be good.
I must have added it by mistake. I do remember having to add Machamp a second time :sweat_smile:

**Review by Aegyo — APPROVED** [review]

**Pull request merged**